### PR TITLE
Update comments to nekojira

### PR DESCRIPTION
**Pull request by Polfo**
I basically merged the best parts of README.md and README.txt
I tried to respect the original texts, and introduced some readability enhancements.
The github difference display is dramatic, sorry for that. My local difference editors show less dramatic differences. 

A bug fix  3e6f4fa and a few enhancements also included.
You may like the password missing display 2d5708e :-)

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -1,26 +1,41 @@
-### 1.3.0 (05 feb 2015)
```
Comment (Polfo) [inline ChangeLog.md]: used the == wordpress format now

File: README.md [hunk 2]
```diff
@@ -1,37 +1,123 @@
-#WP PHP Console
+# === WP PHP Console ===
```
Comment (Polfo) [inline README.md]: this file is now the source of README.txt
only a few changes a required, e.g. removing the # github markdown in wordpress titles

File: README.md [hunk 3]
```diff
@@ -1,37 +1,123 @@
...
+
+### = Options =
+
+In `Settings` `WP PHP Console` page
+
+- You can tick a checkbox to force PHP Console to a SSL connection (of course then if you don't actually have SSL (https), PHP Console simply won't work).
+
+- You can secure your server by specifying IP addresses to restrict the accessibility from the Eval Terminal
+(a single address eg. `192.168.0.4` or an address mask eg. `192.168.*.*` or multiple IPs, comma separated `192.168.1.22,192.168.1.24,192.168.3.*`).
+In case of having issues connecting with the Remote PHP Eval Terminal, temporarily leave this blank.
```
Comment (Polfo) [inline README.md]: Since the bug fix, using * does not work. In my tests, *.*.*.* , my IP and * work, but * not.

File: README.txt [hunk 4]
```diff
@@ -8,33 +8,74 @@ Stable tag: 1.3.0
 License: GPLv2 or later
 License URI: http://www.gnu.org/licenses/gpl-2.0.html
 
-An implementation of PHP Console for WordPress. Use Chrome Dev Tools to debug your WordPress installation!
+An implementation of PHP Console as a WordPress plugin.
+Use Chrome Dev Tools to debug your WordPress installation!
 
 == Description ==
 
-WP PHP Console allows you to handle WordPress PHP errors & exceptions, dump variables, execute PHP code remotely and many other things using Google Chrome extension PHP Console and PhpConsole server library.
```
Comment (Polfo) [inline README.txt]: Both "PHP Console" and "PhpConsole" were used to denote the same. So I chose PHP Console in the documentation.

File: README.md [hunk 5]
```diff
@@ -1,37 +1,123 @@
-#WP PHP Console
```
Comment (Polfo) [inline README.md]: I tried to stick to these names in README.md and README.txt
* PHP Console (by barbushin)
* PHP Console server library
* Chrome plugin PHP Console
* WP PHP Console (WP Plugin)
* Remote PHP Eval Terminal
* JavaScript console
* PHP Console eval & options form

**Comment by nekojira** [comment]
thanks

WordPress and Github use different "flavours" of Markdown

there are MD editors you could use to preview how text appears once rendered:
http://markdownpad.com/ (windows)
http://macdown.uranusjr.com/ (mac)
http://25.io/mou/ (mac)
http://www.texts.io/ (mac)

and I guess many more

**Comment by Polfo** [comment]
Thanks for the improvements.

**Pull request merged**